README.md

```diff
@@ -45,3 +59,8 @@ Then update the package list and install `radicle-cli`:
 
     brew tap radicle/cli https://seed.alt-clients.radicle.xyz/radicle-cli-homebrew.git
     brew install radicle-cli
+
+## Support
+
+* Ask a question in the [Radicle support channel](https://discord.com/channels/841318878125490186/841342872082579466)
```

Comment:
Shall we link Matrix instead / alongside Discord?

Reply:
I can't comment on the discussion channels, but what about linking to our issue tracker?  The README can be copied around into forks or included in OS packages.  Having a link to it may be helpful.

Reply:
I'll create a separate PR for the issue tracker link after this, it shouldn't hold this set of changes up.